docs/troubleshooting.md

```diff
@@ -0,0 +1,8 @@
+# Troubleshooting
+
+## Running into errors
+
+If your setup isn't working and you can not figure out the reason why, running
+`docker compose down -v` will remove the relevant containers and volumes,
+allowing you to run `docker compose up` again from a clean slate.
+
```

Comment:
The remaining trouble shooting hints are dropped?

>  - Sometimes it might help: docker pull ghcr.io/juliushaertl/nextcloud-dev-php74:latest
> - In extreme cases, clean everything: docker system prune --all